Unity-Technologies/ScriptableRenderPipeline
# Hdrp/staging

### Purpose of this PR
- Change the title of the material update progress bar from "Updating Dependent materials" to "Checking material dependencies" (#5041)
- Add specific settings for RTX2080 test projects (#4908)
- Added a multiplier for transmission for directional lights (#5040)
- Hdrp/fix whitelisted stylesheet error (#5038)
- Fix nullrefs in the material editor when entering play mode (#5037)
- Rework XR automated tests for HDRP with XRLayout API (#5036)
- Hdrp/fix/unlit double sided settings view (#5026)
- Invalidating the history for directional, point and spot lights for ray traced screen space shadows when the light has moved (#5025)
- Fix issue with contact shadow when max distance is close to far clip plane (#5029)
- Fixing the flickering in scene and game view for ray tracing (#5035)
- SSS shadergraph preview (#5054)
- Custom pass fixes 3 (#5024)
- Changing the behavior for ray tracing in case a submesh has both opaque and transparent sub meshes (#5005)
- Hdrp/fix camera with prefab override workflow (#4987)
- Improve history buffer management (#4996)
- Update 2103_GI_BakeMixed.png
- Automated Playmode tests (#4977)
- Fixing a bug related to denoising multiple light ray trace shadows (#5057)
- Sync shadergraph settings name (#5055)

---
### Testing status

**Manual Tests**: What did you do?
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/HDRP%252Fstaging/.yamato%252Fupm-ci-abv.yml%2523all_project_ci_fast-trunk/752274/job/pipeline

Any test projects to go with this to help reviewers?

Yamato is green, the failing test on GI come from the engine (see Yamato master), this PR update the reference screenshots but haven't relaunch yamato test to avoid wasting resources.

---
### Comments to reviewers
Notes for the reviewers you have assigned.
